packages/@o3r/design/schematics/extract-token/index.ts

```diff
@@ -46,7 +46,7 @@ function extractTokenFn(options: ExtractTokenSchematicsSchema): Rule {
 
   return async (tree, context) => {
     try {
-      // eslint-disable-next-line @typescript-eslint/naming-convention
+      // eslint-disable-next-line @typescript-eslint/naming-convention -- false positive when using dynamic import
```

Comment:
That's a bit annoying actually :S.
Do we have this issue on all Class dynamically imported ? Is it possible to have exception for this case (which seems quite common) ?

Reply:
We can have a look if it's possible to detect them with this rule 
this rule seems to accept a lot of configurations